dbms/src/AggregateFunctions/QuantileExact.h

```diff
@@ -81,7 +81,7 @@ struct QuantileExact
 
     /// Get the `size` values of `levels` quantiles. Write `size` results starting with `result` address.
     /// indices - an array of index levels such that the corresponding elements will go in ascending order.
-    void getMany(const Float64 * levels, const size_t * indices, size_t size, Value * result)
+    virtual void getMany(const Float64 * levels, const size_t * indices, size_t size, Value * result)
```

Comment:
I don't want to pay for extra 8 bytes.

Reply:
The existence of at least one virtual method will add extra 8 bytes to the sizeof struct.

Reply:
Actually functions `getFloat` and `getManyFloat` were unused and throwing exceptions. One solution is to delete them as I did in a latest commit.